### ISSUE TITLE
Amiri font not found in "make doc"

### ISSUE THREAD

**Issue by khaledhosny** [aliftype/amiri#58]
When running `make doc` on a system without an installed version of Amiri, but the Amiri ttf-files created by `make` are present.

The file `documentation-arabic.pdf` is created without using the actual Amiri files; instead the pages are mostly blank.

I was able to create a reasonably looking pdf after applying these changes:

`````` ~~
diff --git a/documentation/documentation-sources/documentation-arabic.tex b/documentation/documentation-sources/documentation-arabic.tex
index 681b626..ea5e2bb 100644
--- a/documentation/documentation-sources/documentation-arabic.tex
+++ b/documentation/documentation-sources/documentation-arabic.tex
@@ -31,9 +31,9 @@
 \setotherlanguage{english}
 \rightfootnoterule

-\setmainfont               [Ligatures=TeX]                     {Amiri}
+\setmainfont               [Ligatures=TeX]                     {amiri-regular.ttf}
 \setmonofont               [Scale=MatchLowercase]              {DejaVu Sans Mono}
-\newfontfamily\arabicfont  [Script=Arabic,Numbers=Proportional]{Amiri}
+\newfontfamily\arabicfont  [Script=Arabic,Numbers=Proportional]{amiri-regular.ttf}
 \newfontfamily\arabicfonttt[Script=Arabic,Scale=MatchLowercase]{DejaVu Sans Mono}

 \newcommand\addff[1]{\addfontfeature{RawFeature={#1}}} % add feature
@@ -44,7 +44,7 @@
 %setlength{\emergencystretch}{3em}  % prevent overfull lines
 \setcounter{secnumdepth}{0}

-\newfontfamily\titlefont[Script=Arabic]{Amiri}
+\newfontfamily\titlefont[Script=Arabic]{amiri-bold.ttf}

 \titleformat*{\section}{\Large\titlefont\color{titlecolor}}
 \titleformat*{\subsection}{\large\titlefont\color{titlecolor}}
```~~

I'm not very confident with TeX yet, so I don't even know if this produces the intended output. At least it produces some output and it uses the files created by `make`.

This could also be a XeTeX configuration issue, I guess.

Original comment by: graphicore

Original Ticket: [amiri/bugs/64](https://sourceforge.net/p/amiri/bugs/64)
``````

**Comment by khaledhosny** [comment]
This won’t handle bold and italic fonts, but I pushed a more complete fix in 6edb152.

Original comment by: khaledhosny

**Comment by khaledhosny** [comment]
- **status**: open --> closed
- **assigned_to**: Khaled Hosny

Original comment by: khaledhosny
